README.md

```diff
@@ -143,6 +173,21 @@ gnfd-cmd -c config.toml head-obj gnfd://bucket-name/object-name
 // head Group
 gnfd-cmd -c config.toml head-group gnfd://groupname
 ```
+
+#### Policy Operations
+```
+// The object policy actions can be create, delete, copy, get or execute
+// It can be set with combined string like "create,delete" by --actions
+// The object policy effect can set to be allow or deny by --effect
+
+// Put object policy to group:
```

Comment:
```suggestion
// Grant "put object" permission to a group
```

Reply:
changed to  "grant object operation permissions to a group",  it not "put object" , the operations can be set by actions